cardano-cli/src/Cardano/CLI/EraBased/Options/Query.hs

```diff
@@ -27,6 +28,142 @@ import qualified Options.Applicative as Opt
 {- HLINT ignore "Use <$>" -}
 {- HLINT ignore "Move brackets to avoid $" -}
 
+pQueryCmdsTopLevel :: EnvCli -> Parser (QueryCmds ConwayEra)
+pQueryCmdsTopLevel envCli =
+  let parsers =
+        [ pProtocolParams envCli
+        , pTip envCli
+        , pStakePools envCli
+        , pStakeDistribution envCli
+        , pStakeAddressInfo envCli
+        , pUTxO envCli
+        , pLedgerState envCli
+        , pProtocolState envCli
+        , pStakeSnapshot envCli
+        , pPoolParams envCli
+        , pLeadershipSchedule envCli
+        , pKesPeriodInfo envCli
+        , pPoolState envCli
+        , pTxMempool envCli
+        , pSlotNumber envCli
+        ]
+      i =
+        Opt.progDesc $
+          mconcat
+            [ "Node query commands. Will query the local node whose Unix domain socket is "
+            , "obtained from the CARDANO_NODE_SOCKET_PATH environment variable."
+            ]
+   in subParser "query" $ Opt.info (asum parsers) i
+
+pProtocolParams :: EnvCli -> Parser (QueryCmds era)
+pProtocolParams envCli =
+  subParser "protocol-parameters" $
+    Opt.info (pQueryProtocolParametersCmd envCli) $
+      Opt.progDesc "Get the node's current protocol parameters"
+
+pTip :: EnvCli -> Parser (QueryCmds ConwayEra)
+pTip envCli =
+  subParser "tip" $
+    Opt.info (pQueryTipCmd ShelleyBasedEraConway envCli) $
+      Opt.progDesc "Get the node's current tip (slot no, hash, block no)"
```

Comment:
```suggestion
      Opt.progDesc "Get the node's current tip (slot number, hash, block number)"
```

Optional.

Rationale: I think abbreviations can be misleading, in particular to non-native.